static/js/drafts.js

```diff
@@ -325,6 +325,10 @@ exports.launch = function () {
 
     function setup_event_handlers() {
         $(".restore-draft").on("click", function (e) {
+            if (click_handlers.is_drag()) {
```

Comment:
We still don't want `drafts.js` to have a dependency on `click_handlers.js`.  We can just use this idiom directly in both files: `    return document.getSelection().type === "Range";`.

If we find any nuances while testing this that makes it more complicated for some reason, we can consider moving the common code to `ui_util.js`.

Reply:
0kay. I'll remove the is_drag function and directly use the idiom for now.